Global/Eclipse.gitignore

```diff
@@ -49,3 +49,6 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Scala IDE specific (Scala & Java development for Eclipse)
```

Comment:
@sherzodv let's keep changes to one file at a time - feel free to link related PRs so everyone can follow along